### ISSUE TITLE
marker is never status SPIDERFIABLE?

### ISSUE THREAD

**Issue by netzpixel** [copiri-six/AME_Spiderfier#1]
First of all, thanks for making this AdvancedMarkerElement compatible :)

The only thing that isn't working for me now, is that I can't set a different icon if a marker is "SPIDERFIABLE" because it never has that status.

I'm trying something like this:

`oms.addListener('format', (marker: any, status: any) => {
      marker.content = parser.parseFromString(
        status === window.OverlappingMarkerSpiderfier.markerStatus.SPIDERFIABLE
          ? plusSvgString
          : pinSvgString,
        'image/svg+xml'
      ).documentElement
    })`

It is also possible I'm doing something wrong, but if I try to use the status "SPIDERFIED" instead, it changes the icon.

**Comment by remcokalf** [comment]
@netzpixel I don't think this is a bug. It is [noted in the docs](https://github.com/jawj/OverlappingMarkerSpiderfier?tab=readme-ov-file#construction) but I also found this confusing. 

The available statuses depend on the `basicFormatEvent` option. If it is `true` you have these 2 statuses:

```
// Marker status values with basicFormatEvent = true
OverlappingMarkerSpiderfier.markerStatus.SPIDERFIED
OverlappingMarkerSpiderfier.markerStatus.UNSPIDERFIED
```

And confusingly, the above two status values are only set _after_ clicking a marker. So a custom unspiderfied icon will only show after clicking an already spiderfied icon, not before clicking it.

If you set `basicFormatEvent` to `false` (the default), you have these 3statuses, with which you can set a basic icon, a spiderfiable icon and a spiderfied icon:

```
// Marker status values with basicFormatEvent = false
OverlappingMarkerSpiderfier.markerStatus.SPIDERFIED
OverlappingMarkerSpiderfier.markerStatus.SPIDERFIABLE
OverlappingMarkerSpiderfier.markerStatus.UNSPIDERFIABLE
```
